app/views/layouts/_nav.html.erb

```diff
@@ -51,5 +59,31 @@
         </div>
       </nav>
     <% end %>
-  </div>
+
+    <nav class="navbar-nav navbar-expand-md">
```

Comment:
Unfortunately, to get the structure to work, I had to repeat the nav slightly differently. Be sure to make your browser window small and then big again to test.